app/js/views/user-login.jsx

```diff
@@ -31,13 +31,13 @@ export default class UserLogin extends React.Component {
     if (this.state.boxes.length > 1) {
       let value = this.state.value !== null ? 0 : this.state.value;
       const options = this.state.boxes.map((box, index) => {
-        if (box.local_ip === this.foxbox.localHostname) {
+        if (box.client === this.foxbox.client) {
```

Comment:
__possible issue:__ what is the ```client``` field? I don't see it anywhere else (e.g. in ```this.foxbox```)?